src/scopes.js

```diff
@@ -8,15 +8,15 @@ module.exports = function(bookshelf) {
   // `bookshelf.knex()` was deprecated in knex v0.8.0, use `knex.queryBuilder()` instead if available
   var QueryBuilder = (bookshelf.knex.queryBuilder) ? bookshelf.knex.queryBuilder().constructor : bookshelf.knex().constructor;
 
-  bookshelf.Model.extend = function(protoProps, constructorProps) {
-    var model = baseExtend.apply(this, arguments);
+  var extended = function(Target) {
```

Comment:
If you look at the signature of Bookshelf `extended` method then you see, that it is the Child "Class" (the result of extending). It felt to me that it would be more self-describing if it will be named this way. But to be hones I named it many times and this is the result.